routes/keyword.js

```diff
@@ -51,15 +51,12 @@ router.get('/:word', (req, res) => {
 })
 
 function incrementKeyword(word, amount) {
-  return new Promise((resolve) => {
+  return new Promise((resolve, reject) => {
     Keyword.findOne({ word })
       .then((keyword) => {
         let oldSum = 0
         if (!keyword) {
-          keyword = new Keyword({
-            word,
-            votes: [],
-          })
+          reject(new Error('Word not found in database'))
```

Comment:
Correct, the keyword can only be voted on if it exists. It should be created when the article is added, if the keyword does not exist.